### PR TITLE
[NodeJS + GraphQL] Upgrade dependencies in sample project

### PR DESCRIPTION
**Pull request by madmath**
Wanted to set-up a site with the Storefront API and ran the `node-graphql-client` sample project. However I found that the dependencies were old. 

I ran the `npm-check-updates` tool (`ncu`), upgrading the dependencies and fixing a few small things that had changed since. I believe I have kept all of the functionality, and I've tested the commands in `package.json`.

I am not an expert in node, nor had I used Babel before. It's quite possible there's newer ways of doing things. If those are pointed out I will gladly modify the code some more!

### CODE REVIEW

File: node-graphql-client/src/graphql-js-client.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+const client = new GraphQLClient(types, {
   url: 'https://graphql.myshopify.com/api/graphql',
   fetcherOptions: {
     headers: {
       'X-Shopify-Storefront-Access-Token': 'dd4d4dc146542ba7763305d71d1b3d38'
     }
   }
 });
+
+module.exports = client;
```
Comment (lemonmade) [inline node-graphql-client/src/graphql-js-client.js:19]: Why did this have to change from a default export?

**Review by lemonmade — APPROVED** [review]
Didn't run it but the code all LGTM, and the approach it's using for compiling the code is just fine even today IMO. It is a bit scary that the generated .js file grew by thousands of lines, and that all presumably gets included in whatever project used this approach, but that's just more a general problem with API wrappers that need runtime type information :/

File: node-graphql-client/src/graphql-js-client.js (hunk 1 above)
Comment (madmath) [reply node-graphql-client/src/graphql-js-client.js]: Tracked this down to https://babeljs.io/docs/en/v7-migration-api#export-changes. Was able to revert back to `export default` but required a change to the way `require` is done.

**Comment by madmath** [comment]
Thanks for the review, made an improvement based on it. Merging!

**Pull request merged**